pyproject.toml

```diff
@@ -13,10 +13,15 @@ requires = [
     "pywinpty (>=0.5);os_name=='nt'",
     "tornado (>=4)",
 ]
+python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*"
 classifiers=[
     "Environment :: Web Environment",
     "License :: OSI Approved :: BSD License",
     "Programming Language :: Python :: 2",
+    "Programming Language :: Python :: 2.7",
     "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.4",
+    "Programming Language :: Python :: 3.5",
+    "Programming Language :: Python :: 3.6",
```

Comment:
As in `testpath`, I don't like using these specific classifiers. They only get outdated and then need to be updated. If we need to specify Python version compatibility, `requires-python` is the way to do it.